multitool/cwd.bzl

```diff
@@ -0,0 +1,24 @@
+"cwd: a rule for executing an executable in the BUILD_WORKING_DIRECTORY"
+
+def _cwd_impl(ctx):
+    output = ctx.actions.declare_file(ctx.label.name)
+    ctx.actions.write(
+        content = "\n".join([
+            "#!/usr/bin/env bash",
+            "execdir=\"$PWD\"",
+            "pushd $BUILD_WORKING_DIRECTORY > /dev/null",
+            "\"$execdir/{tool}\" \"$@\"".format(tool = ctx.file.tool.short_path),
+            "popd > /dev/null",
+            "",
+        ]),
+        output = output,
+    )
```

Comment:
[nit] More of a style preference, but think it'd be a bit more readable to use `ctx.actions.expand_template` here